integration/test_overload_repro.py

```
@@ -0,0 +1,276 @@
+        write_thread.join(timeout=5)
+
+        # Check queue and stats
+        info_after = node0.info("MODULES")
+        q_after = info_after.get("search_query_queue_size", 0)
+        timeouts = info_after.get("search_cancel-timeouts", 0)
+        successful = info_after.get("search_successful_requests_count", 0)
+
+        total_successes = sum(r.get("successes", 0) for r in query_results)
+        total_timeouts = sum(r.get("timeouts", 0) for r in query_results)
+
+        print(f"\n=== WITHOUT FIX ===")
```

Ruff (0.15.21)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)